### PR TITLE
Adding BSR reflection docs for gRPC

### PR DESCRIPTION
**Pull request by perezd**
### Summary
Adding documentation updates for gRPC's new BSR reflection feature.

### Reason
The docs don't cover the new functionality that was introduced in 9.0

### Testing
Proof read the docs.

### CODE REVIEW

File: docs/insomnia/grpc.md [hunk 1]
```diff
@@ -37,6 +37,22 @@ Click **Add Directory** to add multiple at once.
...
+
+**Available in Insomnia version 2024.9.0 and later.**
+
+Insomnia supports [the Buf Schema Registry](https://buf.build/docs/bsr/introduction) for reflection as of version 2024.9.0.
+
+The BSR doesn't require your gRPC servers to expose any reflection endpoints, it's all managed for you externally. You'll need to configure an [API Token](https://buf.build/docs/bsr/authentication) and your [BSR module path](https://buf.build/docs/bsr/module/manage) to get started. See the gRPC request settings for details:
+
+![gRPC request settings dialog](../assets/images/grpc-bsr-settings.png)
+
+Once your request is configured, specify your server host and click the button to the right with a sync icon (it shows "Click to use server reflection" on hover) to fetch your schema information from the BSR. You'll be able to select from the list of known RPCs and send your requset. Request payloads can be hand written using JSON, and the response payloads will be translated into human-readable JSON for you.
```
Comment (akshayjshah) [inline docs/insomnia/grpc.md:50]:
```suggestion
Once your request is configured, specify your server host and click the button to the right with a sync icon (it shows "Click to use server reflection" on hover) to fetch your schema information from the BSR. You'll be able to select from the list of known RPCs and send your request. Request payloads can be hand written using JSON, and the response payloads will be translated into human-readable JSON for you.
```

**Review by akshayjshah — COMMENTED** [review]
One small typo :)

**Comment by akshayjshah** [comment]
@jackkav or @filfreire, would one of you be able to take a look at this?

File: docs/insomnia/grpc.md [hunk 2]
```diff
@@ -37,6 +37,22 @@ Click **Add Directory** to add multiple at once.
 
 "root.proto" imports all other proto files listed in the **Library** directory.
 
+### Use Buf Schema Registry Reflection
+
+**Available in Insomnia version 2024.9.0 and later.**
+
+Insomnia supports [the Buf Schema Registry](https://buf.build/docs/bsr/introduction) for reflection as of version 2024.9.0.
```
Comment (jackkav) [inline docs/insomnia/grpc.md:44]:
```suggestion
**Available in Insomnia version 9.0 and later.**

Insomnia supports [the Buf Schema Registry](https://buf.build/docs/bsr/introduction) for reflection as of version 9.0.
```

**Pull request merged**